hack/make/.integration-daemon-start

```diff
@@ -72,12 +72,6 @@ if [ "$DOCKER_EXPERIMENTAL" ]; then
 fi
 
 dockerd="dockerd"
-if [ -f "/sys/fs/cgroup/cgroup.controllers" ]; then
-	if [ -z "$TEST_IGNORE_CGROUP_CHECK" ] && [ -z "$TEST_SKIP_INTEGRATION_CLI" ]; then
```

Comment:
We should remove this one from the Makefile as well;

- https://github.com/moby/moby/blob/517261761768096087f698808a850dec8e15378e/Makefile#L60